modules/ldsg.js

```diff
@@ -77,33 +87,47 @@ async function processCardAction(int) {
       suggestionIcarus: u.sf.channels.bottestingForum
```

Comment:
might be good to make `u.sf.forums`. `u.sf.channels` is starting to get pretty big